pkg/hubble/parser/threefour/parser_test.go

```diff
@@ -160,6 +161,7 @@ func TestL34Decode(t *testing.T) {
 
 	assert.Equal(t, []string{"host-192.168.60.11"}, f.GetSourceNames())
 	assert.Equal(t, "192.168.60.11", f.GetIP().GetSource())
+	assert.Equal(t, flowpb.TraceReason_ESTABLISHED, f.GetTraceReason())
```

Comment:
I think we still need a test that asserts the value of `f.GetIP.GetSourceXlated()` after parsing.

Reply:
Sure that can't hurt. For context `TestL34Decode` is mostly useful to ensure we're not breaking the `TraceNotify` field layout by accident. It should be parsed as a `TraceNotifyV0` struct which doesn't have `OrigIP` and thus can't have `SourceXlated` set, and this part is already tested in `TestTraceNotifyOriginalIP`.

Reply:
Added `f.GetIP().GetSourceXlated()` checks in `TestL34Decode` :pray: